optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/extended/abstractsolution/TestdataExtendedAbstractSolutionOverridesGetScore.java

```diff
@@ -31,8 +31,9 @@
 @PlanningSolution
 public class TestdataExtendedAbstractSolutionOverridesGetScore extends AbstractSolution<BendableScore> {
```

Comment:
I 'll rename this class so it ends with "Solution" as it's a solution. (Not really related to this PR)